### PR TITLE
added ratepay and updated swish, vipps, dankort icons

### PR DESCRIPTION
**Pull request by maunaw**
** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [x] I have provided a link to the brand icon’s brand guidelines whenever possible.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

**Comment by maunaw** [comment]
@larouxn and @tauthomas01 can you please review these changes and let us know if any improvements.

File: app/assets/images/payment_icons/swish.svg
Comment (tauthomas01) [inline app/assets/images/payment_icons/swish.svg:1]: I see some id attributes using `SVGID_2_` pattern. Can we follow `pi-swish` name convention and convert it?

It could be `pi-swish-radiant-2`

**Review by tauthomas01 — CHANGES_REQUESTED** [review]
Overall, all icons look good.
I left a comment regarding naming convention.

File: app/assets/images/payment_icons/swish.svg
Comment (maunaw) [reply app/assets/images/payment_icons/swish.svg]: Hi, I have updated the swish id attributes. Please check and let us know. Thank you.

**Review by tauthomas01 — APPROVED** [review]
Looks great. Thank you for contributing.

**Comment by maunaw** [comment]
> Looks great. Thank you for contributing.

Thank you. When will it be released as payment icons for shopify?

**Comment by tauthomas01** [comment]
I can't merge the PR due to Github's Actions CI not responding. It seems like Github is experimenting some issues so we'll keep you posted.

https://www.githubstatus.com/incidents/zbpwygxwb3gw?utm_ts=1621149460

**Comment by maunaw** [comment]
> I can't merge the PR due to Github's Actions CI not responding. It seems like Github is experiencing some issues so we'll keep you posted.
> 
> https://www.githubstatus.com/incidents/zbpwygxwb3gw?utm_ts=1621149460

Sure, Thank you! Keep posted.

**Comment by larouxn** [comment]
Agreed with @tauthomas01, the icons look good but we'll have to wait for GitHub Actions CI to run before merge. 🙇‍♂️

**Comment by larouxn** [comment]
CI is back up and running. Looks like the following test is [failing](https://github.com/activemerchant/payment_icons/pull/446/checks?check_run_id=2670897858) for these icons.

https://github.com/activemerchant/payment_icons/blob/c7a2610ca16251de7dceea90c7fbe301877ad9aa/test/unit/payment_icon_test.rb#L61-L105

**Comment by maunaw** [comment]
> CI is back up and running. Looks like the following test is [failing](https://github.com/activemerchant/payment_icons/pull/446/checks?check_run_id=2670897858) for these icons.
> 
> https://github.com/activemerchant/payment_icons/blob/c7a2610ca16251de7dceea90c7fbe301877ad9aa/test/unit/payment_icon_test.rb#L61-L105

I could not get specific error related to these test, neither message is getting displayed. still I have updated title for ratepay svg. 
Please check and let us know.

File: db/payment_icons.yml
Comment (larouxn) [inline db/payment_icons.yml:293]: I believe I found the issue. This `-` is indented one space. We need to remove this indentation.

Comment (larouxn) [inline db/payment_icons.yml:293]: I believe this is the CI issue. This `-` is indented one space. We need to remove this indentation.

**Review by larouxn — CHANGES_REQUESTED** [review]

File: db/payment_icons.yml
Comment (maunaw) [reply db/payment_icons.yml]: @larouxn Thank you. Removed the space. I hope this will work now. Please review and suggest.

**Comment by maunaw** [comment]
> 👏 👏 👏 👏 👏

Thank you so much. When can we expect these icons available in shopify?

**Comment by larouxn** [comment]
> Thank you so much. When can we expect these icons available in shopify?

Can't make any guarantees but my estimate is by end of next week.

**Comment by larouxn** [comment]
> > 👏 👏 👏 👏 👏
> 
> Thank you so much. When can we expect these icons available in shopify?

Update, @maunaw, looking more like next week. 🙇‍♂️

**Comment by maunaw** [comment]
> > > 👏 👏 👏 👏 👏
> > 
> > 
> > Thank you so much. When can we expect these icons available in shopify?
> 
> Update, @maunaw, looking more like next week.

Thank you for update. Please let us know once updated in shopify.

**Comment by larouxn** [comment]
👋 @maunaw, the icons should be available everywhere now.

**Comment by maunaw** [comment]
> 👋 @maunaw, the icons should be available everywhere now.

Thank you so much. Yes, indeed they are available in Shopify. 👋

**Pull request merged**
